test/unit/calculators/checks_benefits_financial_support_calculator_test.rb

```diff
@@ -814,6 +814,24 @@ class CheckBenefitsFinancialSupportCalculatorTest < ActiveSupport::TestCase
         end
       end
 
+      context "#eligible_for_winter_fuel_payment?" do
+        context "when eligible" do
+          should "be true if over state pension age" do
+            calculator = CheckBenefitsFinancialSupportCalculator.new
+            calculator.over_state_pension_age = "yes"
+            assert calculator.eligible_for_winter_fuel_payment?
+          end
+        end
+
+        context "when ineligible" do
+          should "be false if UNDER state pension age" do
```

Comment:
These capitals seem a bit surprising.

I'm also wondering if the statement is true. Isn't that you're equal or below given the assertion is based on being over state pension age?